### ISSUE TITLE
"expected struct `css_in_rust::style::Style`, found enum `std::result::Result`"

### ISSUE THREAD

**Issue by nugetchar** [lukidoescode/css-in-rust#9]
When trying to play with the example, I get the following error:

`expected struct 'css_in_rust::style::Style', found enum 'std::result::Result'`

My `Cargo.toml` is as following:

```toml
[dependencies]
log = "0.4"
wasm-logger = "0.2"
wasm-bindgen = "0.2.62"
serde = "1.0"
serde_derive = "1.0"
yew = "0.15"
web-sys = "0.3.39"

[dependencies.css-in-rust]
version = "0.2.2"
features = ["yew_integration", "web_sys", "seed_integration"]
```

And my code is pretty much the same as in the last example: 

```rust
use web_sys::HtmlInputElement;
use yew::prelude::*;
use css_in_rust::*;

pub struct Model {
    link: ComponentLink<Self>,
    node_ref: NodeRef,
    pub style: css_in_rust::Style,
}

pub enum Msg {
    SendMessage,
}

impl Component for Model {
    type Message = Msg;
    type Properties = ();

    // https://doc.rust-lang.org/rust-by-example/trait.html
    fn create(_: Self::Properties, link: ComponentLink<Self>) -> Self {
        let style = match css_in_rust::Style::create(
            String::from("Model"), 
            String::from(
                r#"
                    width: 100%;
                    margin: auto;
                    height: 150px;
                    display: flex;
                    max-width: 700px;
                "#,
            ),
        ) {
           // this is where the compilation fails
            Ok(style) => {style},
            Err(error)  => {
                panic!("An error occured while creating the style {}", error);
            }
        };
        Self {
            link,
            node_ref: NodeRef::default(),
            style
        }
    }


    ...
}

```

**Comment by lukidoescode** [comment]
Sorry for coming back to you this late. I need to work on getting notified better.
For some reason I do get the same error in my IDE, but when I compile the code it works perfectly well. I've had such errors in the past with other libraries. At some point (I think it was from 0.1.x to 0.2.0) I updated `css_in_rust::Style::create()` to return a `Result`. In my projects my IDE got confused by this.

Usually deleting the `target`directory at the project root does the trick. Rust will automatically rebuild this directory.

Just for reference: I use VSCode with the rust-analyzer plugin. I got these errors, too. However, it compiles.

Whenever I get it to run there will be a full blown example application with JWT authentication and stuff online as soon as I get it to run properly. This can then serve as a project basis to merge now and then in order to stay on top of versions.
